Cargo.toml

```diff
@@ -47,6 +47,7 @@ xattr = "1.3.1"
 tempfile = "3.9.0"
 rand = { version = "0.8", features = ["small_rng"] }
 bytesize = "1.3.0"
+time = { version = "0.3", features = ["formatting"] }
```

Comment:
would you mind using chrono ? 
we are trying to use only crate for time mgmt

sorry

Reply:
Sure, I've switched to chrono. Also, there is no point in checking the formatting of the cmdline, as it can be empty, a path, or just an executable name.